libs/ui/src/virtual_keyboard.tsx

```diff
@@ -1,275 +0,0 @@
-import React from 'react';
```

Comment:
Duplicate of `libs/ui/src/virtual_keyboard/virtual_keyboard.tsx` that was never removed in a previous refactor.